jupyterlab/tests/mock_packages/test-hyphens-underscore/pyproject.toml

```
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["jupyter_packaging~=0.7.9", "jupyterlab>=3.0.0,==3.*", "setuptools>=40.8.0", "wheel"]
```

We might also want to use:

```suggestion
requires = ["jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"]
```